ktor-server/ktor-server-cio/jvm/src/io/ktor/server/cio/CIOApplicationResponse.kt

```diff
@@ -31,6 +31,9 @@ internal class CIOApplicationResponse(
     @Volatile
     private var chunkedJob: Job? = null
 
+    @Volatile
+    private var sent: Boolean = false
```

Comment:
Could you tell me why the `responded` flag from BaseApplicationEngine can't be reused?

Reply:
reworked solution